core-bundle/src/EventListener/TwoFactorFrontendListener.php

```diff
@@ -121,7 +121,12 @@ public function __invoke(RequestEvent $event): void
             return;
         }
 
-        $targetPath = $this->getTargetPath($request->getSession(), $token->getProviderKey());
+        // Backwards compatibility with scheb/2fa-bundle <5.8
```

Comment:
What's the reason not to simply increase the dependency? Why do we want to remain compatible with a lower version? I understand the check for Symfony <5.2 but for this bundle?

Reply:
Changed in e7be1b561bd67925956d794671849666c1b88948.